stride/problem/acquisitions.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 from collections import OrderedDict
 from cached_property import cached_property
+import mosaic
```

Comment:
Please, could you import mosaic after all of the external packages? Simply to follow the same convention across files.

Reply:
See commit [4561726](https://github.com/trustimaging/stride/pull/30/commits/4561726a69d476df160dfcfe4964b51b81859c37)